dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -243,6 +223,78 @@ def install(self):
                 raise ImportError("Error while importing the DALI TF plugin built from source, will not install")
             print("Installation successful")
 
+    def check_install_env(self):
+        print("Checking build environment for DALI TF plugin ...")
+        print(self.debug_str())
+
+        Path(self.plugin_dest_dir).mkdir(parents=True, exist_ok=True)
+
+        if not self.tf_version or not self.tf_path:
+            error_msg = "Installation error:"
+            error_msg += "\n Tensorflow installation not found. Install `tensorflow-gpu` and try again"
+            error_msg += '\n' + self.debug_str()
+            raise ImportError(error_msg)
+
+    def test(self):
+        from nvidia.dali.pipeline import pipeline_def
+        import nvidia.dali.types as types
+        import nvidia.dali.plugin.tf as dali_tf
+        import tensorflow as tf
+        try:
+            from tensorflow.compat.v1 import Session
+        except Exception:
+            # Older TF versions don't have compat.v1 layer
+            from tensorflow import Session
+
+        try:
+            tf.compat.v1.disable_eager_execution()
+        except Exception:
+            pass
+
+        @pipeline_def()
+        def get_dali_pipe():
+            data = types.Constant(1)
+            return data
+
+        def get_data():
+            batch_size = 3
+            pipe = get_dali_pipe(batch_size=batch_size, device_id=types.CPU_ONLY_DEVICE_ID, num_threads=1)
+            daliop = dali_tf.DALIIterator()
+            out = []
+            with tf.device('/cpu'):
+                data = daliop(pipeline = pipe,
+                              shapes = [(batch_size)],
+                              dtypes = [tf.int32],
+                              device_id = types.CPU_ONLY_DEVICE_ID)
+                out.append(data)
+            return [out]
+
+        test_batch = get_data()
+        with Session() as sess:
+            print(sess.run(test_batch))
+
+    def install(self):
+        self.check_install_env()
+
+        if self.prebuilt_exact_ver:
+            self.install_prebuilt()
+        else:
```

Comment:
```suggestion
        if self.prebuilt_exact_ver:
            self.install_prebuilt()
            try:
                self.test()
                return
            except ImportError as e:
                pass
```

Reply:
Maybe we should test prebuild as well, and if it fails try to build from source.